tests/host_verify/host/host.cpp

```diff
@@ -198,6 +198,25 @@ static int _verify_report(
         _read_binary_file(
             endorsements_filename, &endorsements_data, &endorsements_file_size);
 
+        result = oe_verify_remote_report(
+            report_data, report_file_size, endorsements_data, endorsements_file_size, NULL);
+        if (pass)
+            OE_TEST(result == OE_OK);
+        else
+        {
+            // Note: The failure result code is different between linux vs
```

Comment:
Why is this the case and should that be something that's addressed in the implementation?

Reply:
I’ll file another issue to address this. Thank you.